lib/presentation/widget/destination_picker/destination_picker_viewmodel.dart

```diff
@@ -204,7 +204,7 @@ class DestinationPickerViewModel extends BaseViewModel {
   List<SharedSpaceNodeNested> get currentSharedSpaceList => store.state.destinationPickerState.sharedSpacesList;
 
   bool _isDriveEnable() {
-    return store.state.functionalityState.isDriveEnable();
+    return store.state.functionalityState.isDriveEnabled() || store.state.functionalityState.isSharedSpaceEnabledV5();
```

Comment:
`isSharedSpaceEnabledV5` as I understand it is a condition for both `Drive` and `Workgroup` in the `sharedSpace` screen right?

Reply:
Yes. For back-end `v5`, `Drive` or `Workgroup`  sub functionalities of `SHARED_SPACE`.

Reply:
So is there a way to hide the `drive` in v5

Reply:
The function of `Drive` or `Workgroup` will be controlled by `**_CREATION_RIGHT`. If a `Workgroup` or `Drive` is available before disabling `**_CREATION_RIGHT`, all of these are showed normally.

Reply:
I thought `_CREATION_RIGHT ` was only managing the display of creating a drive or workgroup. As for displaying it on sharedSpace that should be another variable.